### ISSUE TITLE
I got a very, very low accuracy

### ISSUE THREAD

**Issue by lan-sky** [open-mmlab/OpenPCDet#71]
when i want use pretrained model to test. I got this result:
![image](https://user-images.githubusercontent.com/33541823/82456496-e1393600-9ae6-11ea-8158-33614f3e6d90.png)
I think there are some problem in my dataset. But i can train my model. When i use my model to test. I got the same result:
![image](https://user-images.githubusercontent.com/33541823/82456843-5147bc00-9ae7-11ea-8137-a85e3a711be1.png)
I dont know why

**Comment by GuilinZ** [comment]
Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky

**Comment by lan-sky** [comment]
> Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky

Not yet. I used the spconv1.1 before. It doesnt work when I use spconv1.0
If you solve this problem. Tell me how to solve it. THX @GuilinZ

**Comment by GuilinZ** [comment]
> > Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky
> 
> Not yet. I used the spconv1.1 before. It doesnt work when I use spconv1.0
> If you solve this problem. Tell me how to solve it. THX @GuilinZ

Hi I have solved my problem, it is because I didn’t use ‘- - set ‘ setting mentioned in the repo. But while evaluating your own model, you should not use this setting.

**Comment by lan-sky** [comment]
> > > Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky
> > 
> > 
> > Not yet. I used the spconv1.1 before. It doesnt work when I use spconv1.0
> > If you solve this problem. Tell me how to solve it. THX @GuilinZ
> 
> Hi I have solved my problem, it is because I didn’t use ‘- - set ‘ setting mentioned in the repo. But while evaluating your own model, you should not use this setting.
Can you show me the command? I test pointpillar model like this: python3 test.py --cfg_file cfgs/pointpillar.yaml --batch_size 4 --ckpt pointpillar.pth

**Comment by GuilinZ** [comment]
> > > Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky
> > 
> > 
> > Not yet. I used the spconv1.1 before. It doesnt work when I use spconv1.0
> > If you solve this problem. Tell me how to solve it. THX @GuilinZ
> 
> Hi I have solved my problem, it is because I didn’t use ‘- - set ‘ setting mentioned in the repo. But while evaluating your own model, you should not use this setting.
> Can you show me the command? I test pointpillar model like this: python3 test.py --cfg_file cfgs/pointpillar.yaml --batch_size 4 --ckpt pointpillar.pth

I used exactly the same command as in the repo to test the PartA2 model.
python test.py --cfg_file cfgs/PartA2_car.yaml --batch_size 4 --ckpt PartA2.pth \ 
    --set MODEL.RPN.BACKBONE.NAME UNetV0 MODEL.RPN.RPN_HEAD.ARGS use_binary_dir_classifier:True

**Comment by GuilinZ** [comment]
> > > > Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky
> > > 
> > > 
> > > Not yet. I used the spconv1.1 before. It doesnt work when I use spconv1.0
> > > If you solve this problem. Tell me how to solve it. THX @GuilinZ
> > 
> > 
> > Hi I have solved my problem, it is because I didn’t use ‘- - set ‘ setting mentioned in the repo. But while evaluating your own model, you should not use this setting.
> > Can you show me the command? I test pointpillar model like this: python3 test.py --cfg_file cfgs/pointpillar.yaml --batch_size 4 --ckpt pointpillar.pth
> 
> I used exactly the same command as in the repo to test the PartA2 model.
> python test.py --cfg_file cfgs/PartA2_car.yaml --batch_size 4 --ckpt PartA2.pth \
> --set MODEL.RPN.BACKBONE.NAME UNetV0 MODEL.RPN.RPN_HEAD.ARGS use_binary_dir_classifier:True

It seems that your problem is not on this setting, since the author said only PartA2 model needs to specify the - - set parameter

**Comment by lan-sky** [comment]
> > > > > Hi, I have the same problem here, did you figure it out? Thanks! @lan-sky
> > > > 
> > > > 
> > > > Not yet. I used the spconv1.1 before. It doesnt work when I use spconv1.0
> > > > If you solve this problem. Tell me how to solve it. THX @GuilinZ
> > > 
> > > 
> > > Hi I have solved my problem, it is because I didn’t use ‘- - set ‘ setting mentioned in the repo. But while evaluating your own model, you should not use this setting.
> > > Can you show me the command? I test pointpillar model like this: python3 test.py --cfg_file cfgs/pointpillar.yaml --batch_size 4 --ckpt pointpillar.pth
> > 
> > 
> > I used exactly the same command as in the repo to test the PartA2 model.
> > python test.py --cfg_file cfgs/PartA2_car.yaml --batch_size 4 --ckpt PartA2.pth 
> > --set MODEL.RPN.BACKBONE.NAME UNetV0 MODEL.RPN.RPN_HEAD.ARGS use_binary_dir_classifier:True
> 
> It seems that your problem is not on this setting, since the author said only PartA2 model needs to specify the - - set parameter
Yes, I cant get the result from every model.

**Comment by sshaoshuai** [comment]
@lan-sky 
For running inference with the provided `PartA2_car.pth`, please run the following command: 
```
python test.py --cfg_file cfgs/PartA2_car.yaml --batch_size 4 --ckpt PartA2.pth \ 
    --set MODEL.RPN.BACKBONE.NAME UNetV0 MODEL.RPN.RPN_HEAD.ARGS use_binary_dir_classifier:True
```

But for any other provided models or the models you trained by yourself, you DO NOT need to add the `--set xxx` option.

**Comment by lan-sky** [comment]
> @lan-sky
> For running inference with the provided `PartA2_car.pth`, please run the following command:
> 
> ```
> python test.py --cfg_file cfgs/PartA2_car.yaml --batch_size 4 --ckpt PartA2.pth \ 
>     --set MODEL.RPN.BACKBONE.NAME UNetV0 MODEL.RPN.RPN_HEAD.ARGS use_binary_dir_classifier:True
> ```
> 
> But for any other provided models or the models you trained by yourself, you DO NOT need to add the `--set xxx` option.

Thanks for you reply. I know when to add the --set. But I think it is not the problem here. Maybe there are something wrong in my kitti-dataset

**Comment by Blinjuan** [comment]
> 
> 
> when i want use pretrained model to test. I got this result:
> ![image](https://user-images.githubusercontent.com/33541823/82456496-e1393600-9ae6-11ea-8158-33614f3e6d90.png)
> I think there are some problem in my dataset. But i can train my model. When i use my model to test. I got the same result:
> ![image](https://user-images.githubusercontent.com/33541823/82456843-5147bc00-9ae7-11ea-8137-a85e3a711be1.png)
> I dont know why

**Comment by Blinjuan** [comment]
> 
> 
> when i want use pretrained model to test. I got this result:
> ![image](https://user-images.githubusercontent.com/33541823/82456496-e1393600-9ae6-11ea-8158-33614f3e6d90.png)
> I think there are some problem in my dataset. But i can train my model. When i use my model to test. I got the same result:
> ![image](https://user-images.githubusercontent.com/33541823/82456843-5147bc00-9ae7-11ea-8137-a85e3a711be1.png)
> I dont know why

hi，my problem is different from you，i use command ‘python test.py --cfg_file cfgs/pointpillar.yaml --batch_size 4 --ckpt pointpillar.pth’,and i also get some results
![图片](https://user-images.githubusercontent.com/66398813/83710565-ab9e5c00-a653-11ea-8f51-5a61aeaef245.png)
 I don't know why  i can get so hign accuracy, and i  read the paper” PV-RCNN: Point-Voxel Feature Set Abstraction for 3D Object Detection”，i find the result in paper is lower than me。
is there anything  wrong？

**Comment by sshaoshuai** [comment]
This results of our re-implemented PointPillar is corrected.
